src/meta-srv/src/metasrv.rs

```diff
@@ -248,26 +252,25 @@ impl MetaSrv {
         self.mailbox.clone()
     }
 
+    pub fn procedure_manager(&self) -> ProcedureManagerRef {
```

Comment:
I prefer to return `&ProcedureManagerRef` personally. The API user can choose whether to clone it or call its functions directly.